api/v1alpha1/condition_consts.go

```diff
@@ -106,3 +106,33 @@ const (
 	// is not prepared for VMService consumption.
 	VirtualMachineImageV1Alpha1NotCompatibleReason = "VirtualMachineImageV1Alpha1NotCompatible"
 )
+
+// Condition.Reason for Conditions related to VirtualMachinePublishRequest.
+const (
+	// VirtualMachinePublishRequestSourceVMNotFoundReason (Severity=Error) documents that the source VM is not found.
+	VirtualMachinePublishRequestSourceVMNotFoundReason = "SourceVMNotFound"
+
+	// VirtualMachinePublishRequestSourceVMNotCreatedReason (Severity=Error) documents that the source VM
+	// hasn't been fully created yet.
+	VirtualMachinePublishRequestSourceVMNotCreatedReason = "SourceVMNotCreated"
```

Comment:
Do we have sufficient context to determine between this reason and the UniqueIDNotReady below? Once the VM is created, we have the MoID/UniqueID. What does "fully created" mean in the context here?